oneview-ansible.md

```diff
@@ -5800,8 +5800,8 @@ Retrieve facts about one or more Os Deployment Plans.
 
 | Name          | Description  | Returned | Type       |
 | ------------- |-------------| ---------|----------- |
-| os_deployment_plan_custom_attributes   | Has the editable Custom Attribute facts of the OS Deployment Plans in the Server Profiles format. |  When requested, but can be empty. |  dict |
-| os_deployment_plans   | Has all the OneView facts about the OS Deployment Plans. |  Always, but can be null. |  dict |
+| os_deployment_plan_custom_attributes   | Has the editable Custom Attribute facts of the Os Deployment Plans in the Server Profiles format. |  When requested, but can be empty. |  dict |
```

Comment:
OS? or Os?

Reply:
LGTM except what Sijeesh has already pointed out. It should be OS

Reply:
It has to be changed from the library documentation. Please change it from each of your user stories library documentation and run build.